en/assembly/quick_start_cuav_v5_plus.md

```diff
@@ -66,11 +67,13 @@ The figure below shows how you can access your remote receiver (please find the
 
 ## Spektrum Satellite Receivers
 
-The V5+ has a dedicated DSM cable. The Spektrum satellite receiver should be connected to the flight control DSM/SBUS/RSSI interface.
+The V5+ has a dedicated DSM cable.
+The Spektrum satellite receiver should be connected to the flight control DSM/SBUS/RSSI interface.
```

Comment:
@cuhome - As above, need to confirm you mean "A spektrum ..." rather than "The spektrum"

Reply:
A spektrum